cypress/utils/plugins/index-management-dashboards-plugin/commands.js

```diff
@@ -68,7 +68,7 @@ Cypress.Commands.add('updateManagedIndexConfigStartTime', (index) => {
       'POST',
       `${Cypress.env('openSearchUrl')}/${
         IM_CONFIG_INDEX.OPENDISTRO_ISM_CONFIG
-      }/_update_by_query`,
+      }/_update_by_query/`,
```

Comment:
trailing slash is required for cypress to match client-certificates url correctly.

See [here](https://github.com/cypress-io/cypress/pull/15179/files#diff-78e786f0723a32b827b5af5fb0105ec7f459b33bac183b9ab107299261fd6d70R24-R29) for more info.